test/ejabberd_tests/tests/cassandra_SUITE.erl

```diff
@@ -0,0 +1,66 @@
+%%==============================================================================
+%% Copyright 2016 Erlang Solutions Ltd.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%==============================================================================
+-module(cassandra_SUITE).
```

Comment:
How about moving this test suite to `small` tests? There is no XMPP/HTTP level tests here and that's why I think such a test should be placed in `apps/ejabberd/test` and not here.

In other words, IMO, such internal test cases not touching the end user interface of MongooseIM should be placed in `small` tests.